media-gfx/gnome-photos/metadata.xml

```diff
@@ -5,9 +5,6 @@
 	<email>gnome@gentoo.org</email>
 	<name>Gentoo GNOME Desktop</name>
 </maintainer>
-<use>
-	<flag name="flickr">Ensure runtime Flickr support</flag>
-</use>
```

Comment:
Can't remove this yet. It's still present in the v42.0 ebuild. We'll drop it when we drop the v42.0 ebuild.